datafusion/core/src/physical_plan/joins/symmetric_hash_join.rs

```diff
@@ -458,7 +458,7 @@ impl ExecutionPlan for SymmetricHashJoinExec {
             DisplayFormatType::Default => {
                 let display_filter = self.filter.as_ref().map_or_else(
                     || "".to_string(),
-                    |f| format!(", filter={:?}", f.expression()),
+                    |f| format!(", filter={}", f.expression()),
```

Comment:
This change is so that we don't output all of CastOptions and by extension FormatOptions. It is also more correct to not be printing a debug representation as part of Display